sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -310,6 +311,83 @@ You are not limited to receipts! There are a few prebuilt models to choose from,
 - Analyze invoices using the `prebuilt-invoice` model (fields recognized by the service can be found [here][service_recognize_invoice]).
 - Analyze identity documents using the `prebuilt-idDocuments` model (fields recognized by the service can be found [here][service_recognize_identity_documents]).
 
+### Using Prebuilt Document
+Analyze entities, key-value pairs, tables, styles, and selection marks from documents using the general prebuilt document model provided by the Form Recognizer service.
+Select the Prebuilt Document model by passing `model="prebuilt-document"` into the `begin_analyze_documents` method:
+
+```python
+from azure.ai.formrecognizer import DocumentAnalysisClient
+from azure.core.credentials import AzureKeyCredential
+
+endpoint = "https://<my-custom-subdomain>.cognitiveservices.azure.com/"
+credential = AzureKeyCredential("<api_key>")
+
+document_analysis_client = DocumentAnalysisClient(endpoint, credential)
+
+with open("<path to your document>", "rb") as fd:
+    document = fd.read()
+
+poller = document_analysis_client.begin_analyze_document("prebuilt-document", document)
+result = poller.result()
+
+print("----Entities found in document----")
+for entity in result.entities:
+    print("Entity '{}' has category '{}' with sub-category '{}'".format(
+        entity.content, entity.category, entity.sub_category
+    ))
+    print("...with confidence {}\n".format(entity.confidence))
+
+print("----Key-value pairs found in document----")
+for kv_pair in result.key_value_pairs:
+    if kv_pair.key:
+        print(
+                "Key '{}' found within '{}' bounding regions".format(
+                    kv_pair.key.content,
+                    kv_pair.key.bounding_regions,
+                )
+            )
+    if kv_pair.value:
```

Comment:
Is the actual sample in this order? If not then I think the sample should be changed to this, I cant remember if I updated it I know I thought that this was the right order to showcase things.

Reply:
I just re-ordered because I wanted the output to match the sentence order preceding the sample.

Reply:
I don't think order is important? but I can change the sample if you think it should match

Reply:
Nah it's ok, it's fine as is for now. Was just curious